clang/lib/Sema/TreeTransform.h

```diff
@@ -13830,61 +13820,27 @@ TreeTransform<Derived>::TransformLambdaExpr(LambdaExpr *E) {
     getSema().AddTemplateParametersToLambdaCallOperator(NewCallOperator, Class,
                                                         TPL);
 
-  // Transform the type of the original lambda's call operator.
-  // The transformation MUST be done in the CurrentInstantiationScope since
-  // it introduces a mapping of the original to the newly created
-  // transformed parameters.
```

Comment:
I presume this still applies, right? Could we preserve the comment?

Reply:
I think we could yeah, but it should probably be moved somewhere else (probably into `TemplateInstantiator::TransformFunctionProtoType`) because this code here doesn’t have to care about that anymore after this refactor.

Reply:
Done